stable/pomerium/values.yaml

```diff
@@ -35,83 +33,64 @@ config:
 
 authenticate:
   name: authenticate
-  tls:
-    cert: ""
-    key: ""
-    defaultSANList:
-      []
-      # - example.com
-      # - corp.example.com
-    defaultIPList:
-      []
-      # - 1.2.3.4
-
-  redirectUrl: https://auth.corp.pomerium.io/oauth2/callback
+  redirectUrl: ""
   # see https://www.pomerium.io/docs/identity-providers.html
   idp:
     provider: google
     # url: https://example.oktapreview.com/oauth2/default
     clientID: "REPLACE_ME"
     clientSecret: "REPLACE_ME"
+    serviceAccount: ""
+  tls:
+    cert: ""
+    key: ""
+    defaultSANList: []
+    defaultIPList: []
 
 authorize:
   name: authorize
   tls:
     cert: ""
     key: ""
-    defaultSANList:
-      # - access.corp.example.com
-    defaultIPList:
-      []
-      # - 1.2.3.4
+    defaultSANList: []
+    defaultIPList: []
 
 proxy:
   name: proxy
   tls:
     cert: ""
     key: ""
-    defaultSANList:
-      []
-      # - example.com
-      # - corp.example.com
-    defaultIPList:
-      []
-      # - 1.2.3.4
-  authenticateServiceUrl: https://auth.corp.pomerium.io
-  authorizeServiceUrl: https://access.corp.pomerium.io
-  # authorizeUseInternalUrl:
-  # authenticateUseInternalUrl:
-
-# For any other settings that are optional. for a complete listing see:
-# https://www.pomerium.io/docs/config-reference.html
-extraEnv: {}
-extraArgs: {}
-extraVolumes: {}
-
-image:
-  repository: "pomerium/pomerium"
-  tag: "latest"
-  pullPolicy: "IfNotPresent"
+    defaultSANList: []
+    defaultIPList: []
+  authenticateServiceUrl: ""
+  authorizeServiceUrl: ""
+  authorizeUseInternalUrl: ""
+  authenticateUseInternalUrl: ""
 
 service:
   # Service type can be set to ClusterIP, NodePort or LoadBalancer.
-  type: NodePort
+  type: ClusterIP
   externalPort: 443
   annotations:
-    {}
-    # cloud.google.com/app-protocols: '{"https":"HTTPS"}'
+    # {}
+    # if using GKE's build int load balancer
+    cloud.google.com/app-protocols: '{"https":"HTTPS"}'
+    kubernetes.io/ingress.allow-http: "false"
```

Comment:
@desimone should we impose GKE to the chart user by default?

Reply:
Yeah this was left in from testing thanks. 

Though it's required for GKE, from looking at other charts, it looks like ingress is left as an exercise to the user because they vary so much. I think it would be helpful to keep it commented though.

Reply:
yep, and also we could add some example for nginx-ingress commented out:
```
kubernetes.io/ingress.class: nginx
nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
# authenticate sends big headers or something
nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
```

Reply:
Good call. The buffer size thing is a common issue. 

https://github.com/pomerium/nginx-proxy/commit/0e12ab7ce4f15939db25502d882c6be1990abff9